src/controller/category.js

```diff
@@ -0,0 +1,123 @@
+const { Category } = require("../models");
+
+const getAllCategories = async (req, res) => {
+  // find all categories
+  // be sure to include its associated Products
+
+  try {
+    const data = await Category.findAll();
+    return res.json({ success: true, data });
```

Comment:
I would ask here if sending a `{ success: true }` is necessary - if data comes back, the user will know the fetch was successful, and the downside to doing as you have means you now have data nested under the `data` key. It's not wrong, I would just question whether it is necessary or whether it just adds complexity to your data structure.